modules/cli/src/main/java/com/jaffsoft/swaggerstylevalidator/Main.java

```diff
@@ -47,6 +48,7 @@ private static void printResults(List<StyleError> errorList) {
             for (StyleError error : errorList) {
                 System.out.println(error.toString());
             }
+            System.exit(1);
```

Comment:
I'm not sure about this. The program is successful, but it has found some errors... Was there a reason for you to add this?

Reply:
You could be right. The reason I added this is to make it easier to use during CI tests. If the style validation fails, I want the test to exit with a non-zero so it stops the pipeline. Thoughts?

Reply:
It makes sense. You can keep this code. I will need to check with other project what is the "standard" way of dealing with this in a validator.